### PR TITLE
More accessibility fixes for the license acceptance window. 

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug
Fixes: https://github.com/NuGet/Home/issues/7452  
Regression:   
If Regression then when did it last work:   
If Regression then how are we preventing it in future:   

## Fix
Details: 

Copying from the issue. The ones being fixed are the following: 

### UI:
- [x] Link for "View License" is underlined (it shouldn't because it's not consistent with other license links) (In both, details pane and license acceptance)
- [x] Link for "View License" has a weird color when hovered (In both, details pane and license acceptance)  

After some discussions we decided to a simple modal window instead of expanding the license acceptance window because of the plethora of accessibility issues we have with the current approach.

## Testing/Validation
Tests Added: No  
Reason for not adding tests:  
Validation done:  Manual

### CODE REVIEW

**Comment by nkolev92** [comment]
@NuGet/nuget-client 🔔

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+using System;
+using System.Windows.Automation.Peers;
+using Microsoft.Internal.VisualStudio.PlatformUI.Automation;
+using Microsoft.Internal.VisualStudio.Shell;
+
+namespace NuGet.PackageManagement.UI.Utility
+{
+    // disambiguate from Microsoft.Internal.VisualStudio.PlatformUI.UIElement
+    using UIElement = System.Windows.UIElement;
```
Comment (jainaashish) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs]: nit: add it with other usings

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    public static class AutomationUtilities
+    {
+        const AutomationEvents UndefinedEvent = (AutomationEvents)(-1);
+        static readonly AutomationEvents LiveRegionChangedEvent;
+
+        static AutomationUtilities()
+        {
+            // see if the version of WPF we're running with defines AutomationEvents.LiveRegionChanged
+            if (!Enum.TryParse("LiveRegionChanged", out LiveRegionChangedEvent))
```
Comment (jainaashish) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs]: is this the updated change for modal dialog?

**Review by jainaashish — COMMENTED** [review]
Not sure if this is the updated code to define modal dialog

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs (hunk 2 above)
Comment (nkolev92) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs]: The modal dialog already existed so the change is in. (LicenseFileWindow)
https://github.com/NuGet/NuGet.Client/pull/2525/files#diff-3ef3ba58318bdfbd99c5cf15670dac55R53

This class needs to go away completely as it was part of the other approach. I will remove it.

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs (hunk 1 above)
Comment (nkolev92) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/AutomationUtilities.cs]: This class needs to go I will remove it.

**Review by jainaashish — APPROVED** [review]
LGTM

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml.cs [hunk 3]
```diff
@@ -39,18 +34,20 @@ private void ViewLicense_Click(object sender, RoutedEventArgs e)
             {
                 if (hyperlink.DataContext is LicenseFileText licenseFile)
                 {
-                    if (EmbeddedLicense.Visibility == Visibility.Collapsed)
+                    var window = new LicenseFileWindow()
+                    {
+                        DataContext = licenseFile
+
```
Comment (PatoBeltran) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml.cs]: nit: remove extra line

**Review by PatoBeltran — APPROVED** [review]
The code looks good, I'm curious about the experience... Is it possible to have multiple license windows open at the same time? Should it be possible? If so, can you have multiple of the same?

**Comment by nkolev92** [comment]
@PatoBeltran 
No you cannot have more than 1 open as it's modal, you can only focus on that window. 

I get your point, but at this point, I think it's ok to not be able to view all licenses simultaneously.

**Review by zivkan — COMMENTED** [review]
I don't know anything about WPF or XAML, but it seems good to me.

**Pull request merged**